sampleReact.jsx

```
@@ -1,6 +1,7 @@
 function TestComponent() {
     // ruleid:react-dangerouslysetinnerhtml
     return <div dangerouslySetInnerHTML={createMarkup()} />;
+    <a href='http://external.link' target='_blank'>child</a>
```

Biome

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noreferrer" attribute.


(lint/a11y/noBlankTarget)